docs/book/src/clusterctl/developers.md

```diff
@@ -133,3 +133,45 @@ EOF
   for the kubelet in the management cluster.  
 
 Run `clusterctl init` command provided as output of the local-overrides hack.
+
+### Connecting to a workload cluster on docker
+
+The command for getting the kubeconfig file for connecting to a workload cluster is the following:
+
+```bash
+kubectl --namespace=default get secret/capi-quickstart-kubeconfig -o json \
```

Comment:
Maybe use `jsonpath` since not everyone would have `jq`?

`-o jsonpath={.data.value}`

Reply:
good catch!